istio/tests/src/bookinfo-test.yaml

```diff
@@ -0,0 +1,49 @@
+##################################################################################################
+#  Bookinfo-Test
```

Comment:
what is this for?  this is also uses your own image. won't work for anyone else....

Reply:
ok